ios_mcp/source/menu.c

```diff
@@ -931,6 +1134,15 @@ int menuThread(void* arg)
     // open fsa and mount sdcard
     fsaHandle = IOS_Open("/dev/fsa", 0);
     if (fsaHandle > 0) {
+        // initialize the font
```

Comment:
This could be moved out of the `if (fsaHandle > 0) {` block, since it doesn't rely on FSA.